core_lib/src/structure/layercamera.cpp

```diff
@@ -77,7 +78,15 @@ void CameraPropertiesDialog::setHeight(int height)
 LayerCamera::LayerCamera( Object* object ) : Layer( object, Layer::CAMERA )
 {
     setName(tr("Camera Layer"));
-    viewRect = QRect(QPoint(-400, -300), QSize(800, 600));
+    QSettings settings (PENCIL2D, PENCIL2D);
+    int fieldW = settings.value("FieldW").toInt();
+    int fieldH = settings.value("FieldH").toInt();
+    if (fieldW < 2)
```

Comment:
You should check against both values.
eg...
```suggestion
    if (fieldW < 2 || fieldH < 2)
```